### PR TITLE
Add support for more APIs

### PR DESCRIPTION
**Pull request by pcothenet**
Same as #46, except this time using a branch, not our master, which has evolved into a full on 1.0

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -474,6 +496,12 @@ function Client(opts) {
     call.json = true
     call.qs = call.qs || {};
 
+    call.qsStringifyOptions = {
+      arrayFormat: 'repeat'
+    };
```
Comment (brainflake) [inline lib/client.js:501]: Is there a reason for using this over 'brackets'?

**Comment by brainflake** [comment]
:+1:

File: lib/client.js (hunk 1 above)
Comment (pcothenet) [reply lib/client.js:501]: Let me check this one. We changed it a long time ago, can't quite remember why.

Comment (brainflake) [reply lib/client.js:501]: Yeah, was just curious. I know it shouldn't be 'indices' because we ran into an issue with that.

**Pull request merged**